wiki/Legal/Copyright/en.md

```diff
@@ -4,14 +4,16 @@ legal: true
 
 # osu! Copyright Policy
 
-Last Updated 17th May 2017. [View history here](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Copyright/en.md)
+Last Updated 6th November 2019. [View history here](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Copyright/en.md)
 
 osu! has adopted the following policy concerning copyright infringement in accordance with the Digital Millennium Copyright Act, 17 U.S.C. 512.
 
 The contact information for osu!'s Designated Agent to Receive Notification of Claimed Infringement ("Designated Agent") is listed at the end of this policy, and is on file with the United States Copyright Office ([https://www.copyright.gov/dmca-directory/](https://www.copyright.gov/dmca-directory/)). osu! takes copyright and other intellectual property rights very seriously.
 
 It is osu!'s policy to (1) expeditiously block access to or remove content that it believes in good faith may contain material that infringes the copyrights of third parties and (2) remove and discontinue service to repeat offenders.
 
+Alternatively, osu! is willing to negotiate music licensing via its [Music Licensing](/wiki/Legal/Copyright/Music_Licensing) process. Inquiries may be sent to [music@ppy.sh](mailto:music@ppy.sh).
```

Comment:
I'd kinda like this to be the first line of this document, rather than in the middle of the standard DMCA text.

Something like

"osu! is a heavily community based game, with users uploading levels, often without correctly obtaining permission to use the content contained. We go out of our way to not advertise or profit directly from uploaded content, and are funded by user donations.

A large portion of the donations we receive are used to reinvest in music licensing. You can read more about this process [here](/wiki/Legal/Copyright/Music_Licensing). Licensing enquiries may be sent to [music@ppy.sh](mailto:music@ppy.sh).

Failing that, osu! has adopted the following policy concerning..."

Reply:
adjusted this